pcu/PCU_C.h

```diff
@@ -1,7 +1,21 @@
 #ifndef PCU_C_H
 #define PCU_C_H
 #include "pcu_defines.h"
+#if defined(SCOREC_NO_MPI)
+#include "pcu_pnompi_types.h"
+// Remove MPI calls.
+#define MPI_Init(argc, argv) do { \
+(void) argc; \
+(void) argv; \
+} while (0)
+#define MPI_Finalize(void)
+#ifdef __cplusplus
+extern "C"
```

Comment:
same scope question as in PCU.h

Reply:
Removed these macros and added `#ifndef SCOREC_NO_MPI` to all test cases. Additionally, the change is less obscure to someone casually reading tests as examples.